product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -334,7 +334,11 @@ dataGroups:
       array_nulls: true                        
     backupRetentionPeriod: 30d
 witnessGroups:
-  - region: uksouth
+  - provider: azure
```

Comment:
both data groups must also have `customMaintenanceWindow` fields defined in the example